### PR TITLE
Description boxes

### PR DESCRIPTION
**Pull request by shouston3**
- [x] Add `json` structure for description boxes
- [x] Pull the data through to render on the search page when filters are applied

See #779

### CODE REVIEW

**Comment by shouston3** [comment]
Coverage has been increased in #910 instead

**Comment by shouston3** [comment]
Just need to fix the tests then this should be good to go

**Comment by jamieu** [comment]
This might just be me but when I go direct to the page (via a bookmark) I don't see the boxes? 

Although if I de-select and reselect the filter (or another one) on an already loaded search results page I do see it? Initial load vs AJAX?

- http://localhost:8000/search/museum/science%20museum/gallery/information%20age%20gallery:%20broadcast
- http://localhost:8000/search/museum/science%20museum/gallery/making%20the%20modern%20world%20gallery

File: client/lib/listeners/close-description-box.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module.exports = function () {
+  function select (elem) { return document.querySelector(elem); }
```
Comment (SimonLab) [inline client/lib/listeners/close-description-box.js]: maybe the select function should be define outside the exported function

**Pull request merged**